docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -1,73 +1,131 @@
+Because this platform is meant to be used by the entire organization, by default anyone can contribute and publish to What’s new, without the need for approval.
+Quality assurance is a conversation within and between contributing teams and internal stakeholders, but there are some best practice guidelines described in the last two sections of this topic.
 
-Release notes should be entered into the CMS 2-4 weeks before the feature is available, depending on the size of the product or feature. This gives the GTM team time for promotion and enablement. For Grafana versioned releases, have your content entered in the CMS by the cut-off date communicated by the delivery team. For more information, see the [RADS guidelines](https://grafana-intranet--simpplr.vf.force.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000004oOF7AAM).
+You should enter release notes into the CMS two to four weeks before the feature is available, depending on the size of the product or feature.
+This gives the Go To Market (GTM) team time for promotion and enablement.
+For Grafana versioned releases, have your content entered in the CMS by the cut-off date communicated by the delivery team.
+For more information, refer to the [Record Announce Document Ship (RADS) guidelines](https://grafana-intranet--simpplr.vf.force.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000004oOF7AAM).
 
-Note that in the context of the CMS, the word "published" has a slightly different meaning than in general use:
+{{< admonition type="note" >}}
+In the context of the CMS, the word "published" has a slightly different meaning than in general use:
 
-- **Published** - Your entry is complete and in **Published** status. It's either visible on the external _What's new in Cloud_ page or is going to automatically become visible on the release date.
-- **Live** - Your entry is visible on the _What's new in Cloud_ page.
+- **Published**: Your entry is complete and in **Published** status.
+  It's either visible on the external _What's new in Cloud_ page or is going to automatically become visible on the release date.
+- **Live**: Your entry is visible on the _What's new in Cloud_ page.
+
+{{< /admonition >}}
 
 ### Create a What's new entry
 
 When you’re ready to add a What’s new entry, complete the following steps:
 
 1. Fill out the fields:
 
-   - **FEATURE NAME** - Short headline for the improvement. For example, “Grafana OnCall integration for Alerting.”
-   - **FEATURE RELEASE DATE** - Date and time (UTC) that you want this note to be live. This should also be the feature release date. If the feature is behind a feature toggle and gets rolled out only to a fraction of users the date is when the feature was first available to users (opting in).
+   - **FEATURE NAME**: Short headline for the feature.
+
+     For example, _Grafana OnCall integration for Alerting_.
```

Comment:
With all of these new second lines of content indented at the same level as the parent bullet, the list is less scannable for the field names. It starts to have a "wall of text" feeling for me. Can we indent these second lines?
![image](https://github.com/grafana/writers-toolkit/assets/76437239/a4a66c4a-86f3-4216-b8b0-9fa4fb143a59)

Reply:
Yeah, it does feel that way, although I think it's unrelated to my changes. It looks quite wall-y already to me.

Maybe we want a description list instead, I'll try that.

Reply:
Okay leave it. I don't have more suggestions.

Reply:
Maybe a two column table? That's about the only other way I can think of within the limits of markdown...

Reply:
I was also thinking of that, I've got the commit in jdb/2024-02-review-contribute-to-whats-new-table.

<img width="786" alt="Screenshot 2024-02-15 at 09 22 15" src="https://github.com/grafana/writers-toolkit/assets/4599384/f25261ad-be2a-4dbf-bdea-5a2cae58794b">


Personally, it's my favorite so far.

Reply:
yeah that's dope!

Reply:
Three... I meant a three column table :-)

Yea that's a LOT easier to read.